themes-default/slim/package.json

```diff
@@ -126,14 +126,17 @@
       "static/js/vender.js",
       "build/**",
       "gulpfile.js",
-      "static/js/templates/name-pattern.vue"
+      "static/js/templates/**/*.vue"
     ],
     "esnext": true
   },
   "ava": {
     "require": [
+      "babel-register",
+      "babel-polyfill",
```

Comment:
The test is failing because it doesn't have `babel-polyfill`.
```
$ cross-env NODE_ENV=test nyc ava
  Could not resolve required module 'babel-polyfill'
```